full-service/src/config.rs

```diff
@@ -25,45 +25,45 @@ use std::{
     sync::Arc,
     time::Duration,
 };
-use structopt::StructOpt;
+use clap::Parser;
```

Comment:
Nit: alphabetical sorting, empty line between `use` statements and other stuff.

Reply:
fixed